keyboards/supersplit/info.json

```diff
@@ -27,85 +22,85 @@
     "layouts": {
         "LAYOUT_split_6x4_9": {
             "layout": [
-                { "matrix": [0, 0], "x": 0, "y": 1.5 },
-                { "matrix": [1, 0], "x": 1, "y": 1.5 },
-                { "matrix": [2, 0], "x": 2, "y": 0.5 },
-                { "matrix": [3, 0], "x": 3, "y": 0 },
-                { "matrix": [4, 0], "x": 4, "y": 0.7 },
-                { "matrix": [5, 0], "x": 5, "y": 1 },
+                { "x": 0, "y": 1.5 },
```

Comment:
Matrix values should be put back, based on what was configured for `supersplit.h`